README.md

```diff
@@ -1,3 +1,19 @@
 # Archicad Add-On CMake Tools
 
 This repository contains the tools needed for Archicad Add-On compilation. The recommended way to use these tools is to add this code as a submodule to your own repository. See [archicad-addon-cmake](https://github.com/GRAPHISOFT/archicad-addon-cmake) for a usage example.
+
+## APIDevKitLinks.json
+
+There is a configuration file that consists of an object containing key-value pairs, in which the keys are Archicad version numbers, and their respective values are the direct download URLs to the public API Development Kit releases.
+
+## Build script
+
+The repo includes a BuildAddOn.py python script, that handles the building of the Add-Ons. This script takes up to 6 arguments:
+
+- -c, --configFile (mandatory): path to the JSON configuration file
+- -v, --acVersion (optional, but mandatory if --devKitPath is used): a list of Archicad version numbers, that the Add-On is built for. These versions must be present in the APIDevKitLinks file's object keys. When not specified, the script takes all versions specified in the configuration file.
+- -l, --allLocalizedVersions (optional): Toggles creating localized release builds.
```

Comment:
we should mention what will happen if not set (which values of the config file will be used regarding this setting)

Reply:
Fixed